### PR TITLE
Removed broken link of  Benchmark from readme

### LINKED ISSUE #402: Virtual DOM Benchmark link in readme no longer valid
The site describes itself as "obselete".

### PR DESCRIPTION
**Pull request by labtocat**
closes #402

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ performance, small size and all the features listed below.
   * A rich set of hooks available, both per vnode and globally for modules,
     to hook into any part of the diff and patch process.
   * Splendid performance. Snabbdom is among the fastest virtual DOM libraries
-    in the [Virtual DOM Benchmark](http://vdom-benchmark.github.io/vdom-benchmark/).
+    in the Virtual DOM Benchmark.
```
Comment (mightyiam) [inline README.md]:
```
* Splendid performance. Snabbdom is among the fastest virtual DOM libraries.
```

I feel the link should be removed completely.

**Comment by mightyiam** [comment]
Also, a tip: https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords.

File: README.md (hunk 1 above)
Comment (labtocat) [reply README.md]: @ mightyiam fell so, removed.

**Comment by labtocat** [comment]
@mightyiam I have made those changes and also squash in to a single commit. (y)

**Comment by paldepind** [comment]
LGTM :tada:

**Pull request merged**